cypress/fixtures/alibaba/constraint.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "e2e-test-constraint",
+  "spec": {
+    "constraintType": "K8sRequiredLabels",
+    "match": {
+      "kinds": [
+        {
+          "kinds": ["Namespace"],
+          "apiGroups": [""]
+        }
+      ],
+      "labelSelector": {},
+      "namespaceSelector": {}
+    },
+    "parameters": {
+      "rawJSON": "{\"labels\":[\"gatekeeper\"]}"
+    },
+    "selector": {
+      "labelSelector": {}
+    }
+  }
+}

```

Comment:
What's the difference between it and Anexia constraints etc.?

Reply:
actually none, you're right, it is not needed to add it to every single provider. will change